Explain why .tooltipped should be used sparingly (#676)
primer/css: modules/primer-tooltips/README.md

## Implementation and accessibility

> Tooltips as a UI pattern should be your last resort for conveying information because it is hidden by default and often with zero or little visual indicator of its existence.

Reviewer: ```suggestion
Tooltips as a UI pattern should be our last resort for conveying information because it is hidden by default and often with zero or little visual indicator of its existence.
```

> Before you add a tooltip, please consider: Is this information essential and necessary* Can the UI be made clearer? Can the information be shown on the page by default?

Reviewer: ```suggestion
Before adding a tooltip, please consider: Is this information essential and necessary* Can the UI be made clearer? Can the information be shown on the page by default?
```

> **Attention**: we use `aria-label` for tooltip contents, because it is crucial that they are accessible to screen reader users. However, `aria-label` **replaces** the text content of an element in screen readers, so only use `.tooltipped` on elements with no existing text content, or consider using `title` for supplement information.

Reviewer: ```suggestion
**Attention**: we use `aria-label` for tooltip contents, because it is crucial that they are accessible to screen reader users. However, `aria-label` **replaces** the text content of an element in screen readers, so only use `.tooltipped` on elements with no existing text content, or consider using `title` for supplemental information.
```
